marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/MappingStepController.java

```diff
@@ -67,6 +67,14 @@
         return emptyOk();
     }
 
+    @RequestMapping(value = "/{stepName}/references",method = RequestMethod.GET)
+    @ApiOperation(value = "Get mapping references associated with the step")
+    @Secured("ROLE_readMapping")
+    public ResponseEntity<JsonNode> getReferences(@PathVariable String stepName) {
+        JsonNode response = MappingService.on(getHubClient().getStagingClient()).getReferences(stepName);
```

Comment:
No need to change, but I would just one-line this - i.e. there's no value in declaring the JsonNode response variable.